### PR TITLE
fix the clang/gcc warning

### PR DESCRIPTION
**Pull request by ArielG-NV**

### CODE REVIEW

**Review by kaizhangNV — APPROVED** [review]

File: source/slang/slang-capability.cpp [hunk 1]
```diff
@@ -201,10 +201,12 @@ void CapabilitySet::addPermutationsOfConjunctionForEachInContainer(CapabilityAto
 
         if constexpr (keyholeAtomToPermuteWith == CapabilityName::target)
         {
+            SLANG_UNUSED(knownTargetAtom);
```
Comment (jkwak-work) [inline source/slang/slang-capability.cpp:204]: You can move this to line 194 at the beginning of the function and remove the other line

**Review by jkwak-work — APPROVED** [review]
Looks good to me.

Were you able to reproduce it tho?

**Comment by ArielG-NV** [comment]
> Were you able to reproduce it tho?

I can repro the warning in some scenarios, not all.

**Review by csyonghe — APPROVED** [review]

**Review by jkwak-work — APPROVED** [review]
Looks good to me.
But I am little worried that this kind of problem will happen again if we cannot figure out how to reproduce it consistently.

**Pull request merged**